### ISSUE TITLE
Categories are not applied

### ISSUE THREAD

**Issue by COLABORATI** [bordoni/fakerpress#89]
Well, that was disappointing. Categories are not applied to the generated data.

**Comment by COLABORATI** [comment]
No matter what I input or what i try, categories are never applied to new posts.

**Comment by bordoni** [comment]
Yeah that is something that I'm aware, weird bug, doesn't happen for everyone. So I was able to reproduce and I will fix it next monday 😉

**Comment by COLABORATI** [comment]
great, thanks for your attention!

**Comment by COLABORATI** [comment]
With the new release categories seem to be working now, this is GREAT, thank you! 

But when I want to add a second set of taxonomy field rules, things go wrong. 

Imagine posts that should have each one category out of two different groups of categories, say "big, small" and "red, green". Well, I believe you get the idea, from what I understand the interface for fakerpress was made with exactly that possibility in mind - and this is, btw, _brilliant_ - if it just worked...

I am setting up one taxonomy field group with the terms "big, small" from "categories", 100% rate, quantity 1. 

Then I set up another taxonomy field group with the terms "red, green" from "categories", 100% rate, quantity 1. 

The result should be a set of posts that have categories "big & red" or "big & green" or "small & red" or "small & green" - but I get postings that are "big" or "small", but do not have terms from both categories. 

When I set the quantity to "2" then categories get combined that should not be combined, like "big & small". 

Note: with only one taxonomy field set everything works as expected.

BTW there is still that ">" sign between the two quantity input fields - it makes no sense from my point of view - what does "1 bigger than 2" mean in this context? Maybe it is wrong to interpret the ">" in a mathematical way, however it is confusing. Better would be to write the word "to" or add two words: 
"Between [ ] and [ ]"
